metrics/metrics.go

```diff
@@ -726,6 +728,24 @@ func initMetrics(namespace, subsystem string, constLabels prometheus.Labels) {
 			Help:      "Counter of stale read requests bytes",
 		}, []string{LblResult, LblDirection})
 
+	TiKVPipelinedFlushLenHistogram = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "pipelined_flush_len",
+			Help:      "Bucketed histogram of length of pipelined flushed memdb",
+			Buckets:   prometheus.ExponentialBuckets(10000, 2, 13), // 10K ~ 40M
```

Comment:
Super long values may lead to flush batch with keys less than `MinFlushKeys`(10000), what about `prometheus.ExponentialBuckets(1000, 2, 16)`(1K to 32M)?

32M keys should be enough because the MaxFlushSize is 128MB, it's about 4 bytes per KV, even the size of key is longer than 4 bytes.

Reply:
Looks good